### DISCUSSION TITLE
yarn download:plugins  emits Found 27 missing packages:

### DISCUSSION THREAD

**Discussion by cekvenich2** [eclipse-theia/theia#11839] [Q&A]
debian and node 16.18 - just tryinng to build. One step is download plugins

It has this error:

* js-levenshtein
* inversify
* minimist
* native-keymap
* electron
* ajv
* dompurify
* express
* fs-extra
* fuzzy
* lodash.debounce
* lodash.throttle
* markdown-it
* react
* react-dom
* react-virtuoso
* vscode-languageserver-protocol
* vscode-languageserver-types
* vscode-uri
* ws
* requestretry
* unzip-stream
* process
* mkdirp
* tar
* electron-store
* fix-path

Help?

**Comment by msujew** [comment]
Hey @cekvenich2,

the `download:plugins` script shouldn't emit errors related to runtime packages, as it's a CLI script which just downloads some vscode extensions to be consumed in Theia. Can you please make sure to run `yarn install` first and then post the whole log in case the issue persists?
